components/nodemanager-service/pgdb/node-managers-cloud.go

```diff
@@ -324,25 +313,27 @@ func (db *DB) UpdateOrInsertInstanceSourceStateInDb(instance InstanceState, mgrI
 	}
 	var id string
 	connectionErr := fmt.Sprintf("node state is %s", instanceState)
-	switch instance.State {
-	case "terminated":
-		// if the instance state we're getting is terminated, we only want to update the node if we already
-		// have it in the system. it's silly to *add* nodes that are already terminated here.
+	name := instance.Name
+	if len(name) == 0 {
+		name = instance.ID
+	}
+	switch instanceState {
+	case "TERMINATED", "STOPPED":
```

Comment:
I'm confused about why these changed from lowercase to uppercase...

Reply:
i was using instance.State before, but i actually translate instance.State into instanceState via this function earlier up in the function, so I figured I may as well use that: https://github.com/chef/automate/blob/master/components/nodemanager-service/pgdb/node-managers-cloud.go#L350

Reply:
Ahhh, I totally missed that. Thanks for the explanation.

Reply:
Would it make sense to add a default case to the switch that logs/fails with an unrecognized value?